manifest.toml

```diff
@@ -55,10 +55,11 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://downloads.wordpress.org/release/wordpress-6.3.1.zip"
-        sha256 = "8a3c60c390b1e29143ba49292a5e38d8f7bff2ff6b67a3e4988d157f555a0601"
+        url = "https://downloads.wordpress.org/release/wordpress-6.4.1.zip"
+        sha256 = "280d1e4cbbf862ef48b92e28ac495f729536e10b5403c154f8303279c012f69e"
 
     [resources.system_user]
+    allow_email = true
```

Comment:
eeeh okay but without any actual use of the `mail_pwd` setting anywhere this serves no purpose ? x_X